docs/metaed-project/Association/UserLMSSectionAssociation.metaed

```diff
@@ -25,10 +25,10 @@ Association LMSUserLMSSectionAssociation
         role name Enrollment
     date StartDate
         documentation "Month, day, and year of the user's entry or assignment to the section."
-        is required
+        is optional
```

Comment:
Made this change because all three extractors have put nulls for StartDate/EndDate.  Will note in ticket for discussion.

Reply:
Interesting, if none of them support a StartDate or EndDate for a section association, why do we have these columns? Or were we going to infer this? Definitely worth an extra discussion about whether they should be kept.